harness/determined/deploy/aws/templates/lore.yaml

```diff
@@ -0,0 +1,1084 @@
+                master_url: $scheme://local-ipv4:$port
+                max_idle_agent_period: ${MaxIdleAgentPeriod}
+                max_agent_starting_period: ${MaxAgentStartingPeriod}
+                min_instances: ${MinDynamicAgents}
+                max_instances: ${MaxDynamicAgents}
+                spot: ${SpotEnabled}
+                spot_max_price: "${SpotMaxPrice}"
+                network_interface:
+                  public_ip: true
+                  security_group_id: ${AgentSecurityGroup.GroupId}
+                  subnet_id: ${Subnet1}
+                type: aws
+                root_volume_size: 200
+                ssh_key_name: ${Keypair}
+                tag_key: det-${AWS::StackName}
+                tag_value: det-agent-${AWS::StackName}
+                agent_reconnect_attempts: ${AgentReconnectAttempts}
+                agent_reconnect_backoff: ${AgentReconnectBackoff}
+                agent_config_file_contents: ${AgentConfigFileContents}
+                startup_script: |
+                  mkdir -p /mnt/efs
+                  mount -t efs ${EFSVal}:/ /mnt/efs
+                  if [ ! -z "${DockerUser}" ] && [ ! -z ${DockerPass} ]; then
+                    echo "${DockerPass}" | docker login --username "${DockerUser}" --password-stdin
+                  fi
+
+            scheme: $scheme
+            cpu_env_image: ${CpuEnvImage}
+            gpu_env_image: ${GpuEnvImage}
+            notebook_timeout: ${NotebookTimeout}
+            lore_version: ${LoreVersion}
+
+            bind_mounts:
+              - host_path: /mnt/efs
+                container_path: /run/determined/workdir/shared_fs
+            EOF
+
+            apt-get remove docker docker-engine docker.io containerd runc
+            apt-get update
+            apt-get install -y \
+                apt-transport-https \
+                ca-certificates \
+                curl \
+                gnupg-agent \
+                software-properties-common
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+            add-apt-repository \
+               "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+               $(lsb_release -cs) \
+               stable"
+            apt-get update
+            apt-get install -y docker-ce docker-ce-cli containerd.io
+
+            curl -fsSL https://www.amazontrust.com/repository/AmazonRootCA1.pem > \
+              /usr/local/determined/etc/db_ssl_root_cert.pem
+
+            if [ ! -z "${DockerUser}" ] && [ ! -z ${DockerPass} ]; then
+              echo "${DockerPass}" | docker login --username "${DockerUser}" --password-stdin
+            fi
+
+            docker network create determined
+
+            docker stop $(docker ps -a -q)
+            docker rm $(docker ps -a -q)
+
+            mkdir -p /mnt/efs
+
+            date >> /tmp/cloud-init-log
+
+            apt-get install -y binutils git
+            if [ ! -d "efs-utils" ]; then
```

Comment:
why do we need to install `efs-utils` here when the `efs.yaml` doesn't? it's to call `mount -t efs...` right?

Reply:
the agent amis are custom and do this already but for master we don't build the AMIs afaik and we these utilities are missing there.